### ISSUE TITLE
AssetGetImage generating Image contructor with default scale

### ISSUE THREAD

**Issue by bziembanski** [FlutterGen/flutter_gen#272]
```
Image image({
    Key? key,
    AssetBundle? bundle,
    ImageFrameBuilder? frameBuilder,
    ImageErrorWidgetBuilder? errorBuilder,
    String? semanticLabel,
    bool excludeFromSemantics = false,
    double? scale = 1.0,
    double? width,
    double? height,
    Color? color,
    Animation<double>? opacity,
    BlendMode? colorBlendMode,
    BoxFit? fit,
    AlignmentGeometry alignment = Alignment.center,
    ImageRepeat repeat = ImageRepeat.noRepeat,
    Rect? centerSlice,
    bool matchTextDirection = false,
    bool gaplessPlayback = false,
    bool isAntiAlias = false,
    String? package = 'core',
    FilterQuality filterQuality = FilterQuality.low,
    int? cacheWidth,
    int? cacheHeight,
  })
```
Because of this, flutter always uses the smallest image. Is there a way to generate code without this default value?

**Comment by lcdsmao** [comment]
This should be fixed in 4.3.0.
Can you give it a try?

**Comment by bziembanski** [comment]
Yep, that was the solution, thanks!
